tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -84,17 +84,19 @@
 
       .example-holder {
         width: 100%;
+        margin-left: -5px;
+        margin-top: -5px;
```

Comment:
same question with these updateds

Reply:
Added this for full-width feature value rows (was looking strange on small width arrangements)